From 89f06418c0b10ca34445c6072c3d22c1c0080ce6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Pertti=20Kellom=C3=A4ki?= <pertti.kellomaki@nokia.com>
Date: Mon, 30 Aug 2010 10:18:34 +0300
Subject: [PATCH] Changes: use /usr/bin/ruby instead of /usr/bin/ruby1.8 in test scripts.
      The reason for this is that rpmbuild picks up a dependence
      to /usr/bin/ruby1.8 which is not provided by any package.

---
 tests/TestScripts/tc_theming.rb     |    2 +-
 tests/TestScripts/ts_prestartapp.rb |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/tests/TestScripts/tc_theming.rb b/tests/TestScripts/tc_theming.rb
index e48ce17..a63a885 100755
--- a/tests/TestScripts/tc_theming.rb
+++ b/tests/TestScripts/tc_theming.rb
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby1.8
+#!/usr/bin/ruby
 #
 # Copyright (C) 2010 Nokia Corporation and/or its subsidiary(-ies).
 # All rights reserved.
diff --git a/tests/TestScripts/ts_prestartapp.rb b/tests/TestScripts/ts_prestartapp.rb
index 6da6918..79c42c0 100755
--- a/tests/TestScripts/ts_prestartapp.rb
+++ b/tests/TestScripts/ts_prestartapp.rb
@@ -1,4 +1,4 @@
-#!/usr/bin/ruby1.8
+#!/usr/bin/ruby
 #
 # Copyright (C) 2010 Nokia Corporation and/or its subsidiary(-ies).
 # All rights reserved.
-- 
1.7.0.4

